Entities/ACrab.cpp

```diff
@@ -2152,18 +2152,23 @@ void ACrab::Update()
 
     if (m_pJetpack && m_pJetpack->IsAttached())
     {
-        // Start Jetpack burn
-        if (m_Controller.IsState(BODY_JUMPSTART) && m_JetTimeLeft > 0)
-        {
-            m_pJetpack->TriggerBurst();
-            // This is to make sure se get loose from being stuck
-            m_ForceDeepCheck = true;
-            m_pJetpack->EnableEmission(true);
-            // Quadruple this for the burst
-            m_JetTimeLeft -= g_TimerMan.GetDeltaTimeMS() * 10;
-            if (m_JetTimeLeft < 0)
-                m_JetTimeLeft = 0;
-        }
+		if (m_JetTimeTotal > 0) {
+			// Throttle depletes based on throttle range
```

Comment:
What is `throttle range` in this comment? I'm pretty sure you have it so throttle depletes based on the ratio of jetTimeLeft to jetTimeTotal.

Reply:
The jet time ratio only plays a role if the jetpack has custom Min/MaxThrottleRange values assigned to it. I guess this piece of code doesn't really have to run if throttle ranges are at zero.

My bad, I didn't really elaborate on it properly.

Reply:
Comment looks good now, thanks.